examples/quantum_walk.ipynb

```diff
@@ -150,7 +149,7 @@
    "source": [
     "Let's compare the quantum walk with a classical random walk. The classical time evolution operator is $e^{-(\\mathcal{T} - I) t}$ where $\\mathcal{T}$ is the transition matrix of the graph.\n",
     "\n",
-    "We choose as our initial condition $|\\psi(0)> = |0>$, that is the walker starts on the first node. Therefore, due to symmetry, the probability of occupation of all nodes besides $| 0 >$ is the same. "
+    "We choose as our initial condition $\\left| \\psi(0) \\right\\rangle = \\left| 0 \\right\\rangle$, that is the walker starts on the first node. Therefore, due to symmetry, the probability of occupation of all nodes besides $\\left| 0 \\right\\rangle$ is the same. "
```

Comment:
Not me, but some might say the parens should be `\left(` and `\right)`.

Reply:
I'm happy to change it if anyone gets tetchy about it.

Reply:
No one is around to complain 😏

Reply:
when the 🐱's away...

Reply:
Speaking as Local TeXpert, this is fine.